tests/interactor/unit/state_machine/commands/AddLabelCommand.cc

```diff
@@ -0,0 +1,19 @@
+#include "interactor/state_machine/commands/Commands.h"
+#include "interactor/unit/state_machine/commands/CommandTest.h"
+#include "test_utils/operators.h"
+
+class AddLabelCommandTest : public CommandTest {};
+
+TEST_F(AddLabelCommandTest, MustAddLabel) {
+  auto task = task_factory_.GetNextTask();
+  auto id = model_controller_->Add(task).AccessResult();
+  Label label;
+  label.set_name("label1");
+  AddLabelCommand add_label_command{id, label};
+  auto command_result = add_label_command.execute(*model_controller_);
```

Comment:
need to use mock of model_controller to make appropriate unit test

Reply:
will be resolved in another PR